packages/opentelemetry-propagation-utils/src/pubsub-propagation.ts

```diff
@@ -50,15 +50,26 @@ const patchArrayFilter = (
     return newArray;
   };
 
-  Object.defineProperty(messages, 'filter', {
+  Object.defineProperty(messages, "filter", {
     enumerable: false,
     value: patchedFunc,
   });
 };
 
+function isPromise(value: unknown): value is Promise<unknown> {
+  return value != null && typeof (value as any).then == "function";
+}
+
+function Promise_finally<T>(p: Promise<T>, onFinally: () => any): Promise<T> {
```

Comment:
Our convention for function names is camelCase: `promiseFinally`.
Also, could be nice to align the style with the rest of the functions in this package, e.g: 
```js
const promiseFinally<T> = (p: Promise<T>, onFinally: () => any): Promise<T> => {...}
```

Reply:
Is this function being called anywhere?

Reply:
It's not. I've removed it